css/style.css

```diff
@@ -93,908 +91,628 @@ h6 {
 }
 
 /*--------------------*/
-
-@font-face {
-  font-family: "icons";
-  src: url("../fonts/icons.ttf?mplvmj") format("truetype"), url("../fonts/icons.woff?mplvmj") format("woff"), url("../fonts/icons.svg?mplvmj#icons") format("svg");
-  font-weight: normal;
-  font-style: normal;
-  font-display: block;
-}
-
-[class^=_icon-]::before,
-[class*=" _icon-"]::before {
-  /* use !important to prevent issues with browser extensions that change fonts */
-  font-family: "icons" !important;
-  font-style: normal;
-  font-weight: normal;
-  font-variant: normal;
-  text-transform: none;
+body {
+  height: 100%;
```

Comment:
why do you need this property ?

Reply:
I needed this rule at the beginning for convenience, I forgot to remove it at the end